src/sentry/static/sentry/app/views/releases/detail/organization/index.jsx

```diff
@@ -0,0 +1,110 @@
+import DocumentTitle from 'react-document-title';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import SentryTypes from 'app/sentryTypes';
+import {t} from 'app/locale';
+import Alert from 'app/components/alert';
+import LoadingError from 'app/components/loadingError';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import withOrganization from 'app/utils/withOrganization';
+import space from 'app/styles/space';
+
+import ReleaseHeader from '../shared/releaseHeader';
+import {getRelease} from '../shared/utils';
+
+class OrganizationReleaseDetails extends React.Component {
```

Comment:
We (as a group) should figure out a consistent naming scheme. The problem with this is if I see `OrganizationReleaseDetails` used somewhere, I generally try to use a fuzzy file finder to open up `OrganizationReleaseDetails`, which may miss this since the path `app/views/releases/detail/organization/index.jsx` doesn't really match the component name at all.

Reply:
Also would like to see this use `AsyncView`